src/mbgl/programs/program.hpp

```diff
@@ -54,7 +55,7 @@ class Program {
     }
 
     static std::string vertexSource(const ProgramParameters& parameters) {
-        return pixelRatioDefine(parameters) + Shaders::vertexSource;
+        return pixelRatioDefine(parameters) + shaders::vertexPrelude + Shaders::vertexSource;
```

Comment:
OpenGL supports taking multiple strings and compile them to a single shader. This could be a micro optimization (so we don't have to do string concatenation)